src/www/pages/components/buttons/index.njk.md

```diff
@@ -177,6 +177,62 @@ Icons should only be used where it is deemed absolutely necessary to communicate
 
 {% markdown %}
 
+## Pay with Swift buttons
+
+<h3>What does it do?</h3>
+
+- Enables users to pay for [West Midlands Cycle Hire](https://www.wmcyclehire.co.uk/) using Swift
+
+<h3>When to use it?</h3>
+
+- When a ticket or fare can be purchased using Swift in a travel operator app
+
+<h3>When not to use it?</h3>
```

Comment:
Convert page to markdown

Reply:
oh that's like that because the title is repeated... so it complains and custom id doesn't work too (old issue already).